### PR TITLE
Fix tests in v7 

### PR DESCRIPTION
**Pull request by sriteja777**
## Proposed changes

1.  linux with node 14.x
    - This was failing due to conflicting `@types/node` versions installed in different packages
    - One solution was to upgrade npm 6 to npm 7
    - The current `package-lock.json` has of version of v2, but npm 6 only supports v1. So maybe that's why upgrading it solved the issue. Also tried deleting the existing package-lock.json and running `npm install` with npm 6, but that didn't solve the issue.
    - `setup-node` doesn't offer any way to specify npm version, so had to upgrade it using command - [related issue](https://github.com/actions/setup-node/issues/213)
2. find element tests
    - These were failing on Linux and Windows.
    - Made the same changes as [you made for v8](https://github.com/webdriverio/webdriverio/commit/33aa5aade1047b1a7876c576a44fe30f1a236ae5). But after this changes Mac tests were failing
    - On debugging found out that, the version of `aria-query` was getting installed as 5.1.3 on linux and windows as 5.2.1 on Mac (mac be due to caching or something). The output of [roleElements](https://github.com/webdriverio/webdriverio/blob/08067cc7d4dd957c892185ad6fed90c4f527944a/packages/webdriverio/src/utils/findStrategy.ts#LL291C9-L291C9) was different in different versions, thereby causing the tests to fail.
    - So had upgraded this package version to 5.2.1 to be consisitent

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by sriteja777** [comment]
Bull still the windows tests are failing at [deep selectors](https://github.com/webdriverio/webdriverio/blob/08067cc7d4dd957c892185ad6fed90c4f527944a/e2e/element.test.js#L289). Wasn't able to find the cause of this. Tried to reproduce this with a test, but that was failing with invalid selector

@christian-bromann can you help in fixing that windows test that is failing?

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 </p>
 
 <p align="center">
-    Next-gen browser and mobile automation test framework for Node.js.
+    Next-gen browser and mobile automation test work for Node.js.
```
Comment (christian-bromann) [inline README.md:10]: Why this change?

**Comment by christian-bromann** [comment]
> can you help in fixing that windows test that is failing?

The Windows tests have been failing for a bit longer and I am inclined to just ignore it.

File: README.md (hunk 1 above)
Comment (sriteja777) [reply README.md:10]: this was just to make some PR. forgot to remove. will remove

**Comment by sriteja777** [comment]
@christian-bromann In that case does that failing windows test will be a blocker for release?

**Comment by sriteja777** [comment]
Windows tests passed. Don't know how 😆

**Comment by christian-bromann** [comment]
> Windows tests passed. Don't know how 😆

Seems like they are just flaky
